cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -159,13 +166,15 @@ protected boolean removeEldestEntry(Map.Entry<UUID, UserTaskInfo> eldest) {
    *                 returns a completed Future.
    * @param step The index of the step that has to be added or fetched.
    * @param isAsyncRequest Indicate whether the task is async or sync.
+   * @param parameters Parsed parameters from http request.
```

Comment:
Looks like we are explicitly passing `null` in some codepath to this function. Can we explain what it means to pass `null` to this method?

Reply:
We only pass `null` in `UserTaskManager`,and that is because we do not do logging in test.

Reply:
Would it be possible to indicate what passing `null` means in the `JavaDoc`?